client/asset/dcr/dcr.go

```diff
@@ -2534,9 +2536,9 @@ func (dcr *ExchangeWallet) PayFee(address string, regFee, feeRate uint64) (asset
 	if err != nil {
 		return nil, err
 	}
-	// TODO: Evaluate SendToAddress and how it deals with the change output
+	// TODO: Evaluate SendToAddress rpc and how it deals with the change output
 	// address index to see if it can be used here instead.
```

Comment:
> Does this solve the TODO?

It doesn't.

Reply:
I see a new TODO: `TODO: Just use the sendtoaddress rpc since dcrwallet respects locked utxos.` is that talking about the same thing?

Reply:
> I see a new TODO: `TODO: Just use the sendtoaddress rpc since dcrwallet respects locked utxos.` is that talking about the same thing?

Don't think so. This todo is about using the sendToAddress rpc instead of creating the tx and then broadcasting.

Reply:
I think you guys are talking about the same thing now.

But no, the TODO is not resolved here.  There's still the possibility of just using the RPC to author the transaction for us.  I honestly don't remember what I was getting at with the "change output address index", but I think I made that comment.

Reply:
Probably talking about randomization of the change output position. I think dcrwallet randomizes it by default, and we assume it's at `0` below.